create-snowpack-app/app-template-preact/src/App.jsx

```diff
@@ -1,7 +1,7 @@
 import { h } from 'preact';
 import { useState, useEffect } from 'preact/hooks';
-import logo from './logo.png';
-import './App.css';
```

Comment:
I believe both of these need to stay relative, as you‘re not importing global modules. This seems to throw an error in the tests